dotnet/Services/ProductReviewService.cs

```diff
@@ -301,7 +301,7 @@ public async Task<ReviewsResponseWrapper> GetReviews(string searchTerm, int from
 
             string sortQuery = await this.GetSortQuery(orderBy);
 
-            ReviewsResponseWrapper wrapper = await _productReviewRepository.GetProductReviewsMD($"{searchQuery}{sortQuery}{statusQuery}");
+            ReviewsResponseWrapper wrapper = await _productReviewRepository.GetProductReviewsMD($"{searchQuery}{sortQuery}{statusQuery}", Convert.ToString(from), Convert.ToString(to));
```

Comment:
ReviewsResponseWrapper wrapper = await _productReviewRepository.GetProductReviewsMD($"{searchQuery}{sortQuery}{statusQuery}", from.ToString(), to.ToString());
is also valid.